google-cloud-firestore/src/test/java/com/google/cloud/firestore/DocumentReferenceTest.java

```diff
@@ -275,6 +275,18 @@ public void deserializeDocumentReference() throws Exception {
     assertEquals(documentReference, snapshot.getReference());
   }
 
+  @Test
+  public void getFieldWithFieldMask() throws Exception {
+    doAnswer(getAllResponse(SINGLE_FIELD_PROTO))
+        .when(firestoreMock)
+        .streamRequest(
+            getAllCapture.capture(),
+            streamObserverCapture.capture(),
+            Matchers.<ServerStreamingCallable>any());
+    DocumentSnapshot snapshot = documentReference.get(FieldMask.of(FieldPath.of("foo"))).get();
+    assertEquals("bar", snapshot.get("foo"));
+  }
```

Comment:
You capture parameters:

```
            getAllCapture.capture(),
            streamObserverCapture.capture(),
```
Don't you need to check the captured values?

Reply:
Done